referrer-policy/generic/inheritance/iframe-inheritance-javascript-child.html

```diff
@@ -0,0 +1,70 @@
+<!doctype html>
+<title>Referrer Policy: iframes with javascript url reuse referrer policy</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/common/get-host-info.sub.js"></script>
+<script src="resources/make-html-script.js"></script>
+<meta name="referrer" content="origin">
+<div id="log"></div>
+<script>
+[
+  {
+    srcDocPolicy: ``,
+    expected: self.origin + "/"
+  },
+  {
+    srcDocPolicy: `<meta name="referrer" content="no-referrer">`,
+    expected: undefined
+  }
+].forEach(({ srcDocPolicy, expected }) => {
+  promise_test(t => {
+    return new Promise(resolve => {
+      window.addEventListener("message", t.step_func(msg => {
```

Comment:
I don't think we need `step_func` in a `promise_test`, do we?

Reply:
This is what https://web-platform-tests.org/writing-tests/testharness-api.html#promise-tests suggests, but if I remove the wrappers I end up with a harness failure.

Reply:
That was a mistake in the documentation, https://github.com/web-platform-tests/wpt/pull/21269 fixed it.